registry/README.md

```diff
@@ -4,37 +4,37 @@
 
 ### Build Package
 
-Create `azpkg` and upload to a ßstorage account.
-
-```powershell
-PackageGenerator\AzureGalleryPackager.exe -m <path-to-manifest.json> -o <output-dir>
-```
+Use the `make release` command to create `azpkg` file for Docker Container Registry Marketplace Item, and find it in the "_out" folder. Upload the file to a blob container in a storage account, and make sure to change the access policy of the blob to enable anonymous access.
 
 ### Side-load Package
 
-Connect to management endpoint and install package.
+Update admin credential and storage account information in the `gallery.ps1` script, and use it to connect to management endpoint and install package.
```

Comment:
- "Update THE admin credential**s**"
- and **run** it to connect to **the** management ...